### ISSUE TITLE
"wrong credentials" Error

### ISSUE THREAD

**Issue by matt-imgd** [snd/amazon-associate#5]
I'm getting the following error, although I've double checked the username / password and they work in browser.

Error: wrong credentials
    at handlers.(anonymous function) (/node_modules/amazon-associate/src/client.js:74:21)

**Comment by htmathias** [comment]
I got the same.

**Comment by Kinzi** [comment]
@matt-imgd @htmathias Maybe a bit off-topic but how did you get login credentials for https://assoc-datafeeds-eu.amazon.com/datafeed/listReports in the first place?

My normal credentials are not working in the browser (nor with the plugin)....

**Comment by der-robert** [comment]
Same here. Amazon didn't answer my request. They send me only a link where i can reset my partnernet login.

**Comment by Kinzi** [comment]
Yeah support is not very helpful here. They actually kept pointing me to the product API while I kept saying "this feed-url NOT the product API" - it was a trap....

Still: Anyone knows how to get access to this?!?

**Comment by der-robert** [comment]
Yes. Still the same here. Since 1 1/2 weeks i try to explain to the support what i mean. Once the said they need the Access Key to create the creditals. I send them the key. Next answere is,  that i allready have used the amazon product api and whats the point of my question ist. Stupid. Next time they said they are not responsive for the data feed. Contact aws support they said. And now since 1 1/2 they dont understand what i want. ...ahhhhh

**Comment by nadien** [comment]
@der-robert @Kinzi @htmathias @matt-imgd 

Someone fixed the issue?, same here.

**Comment by Kinzi** [comment]
@nadien Nope, I eventually gave up after a long journey from german associates support > AWS support > states associates support > german associates support. Actually the us support seemed to know what I wanted but since it's a  german account they couldn't help me. 

I wonder if this feed actually exists?!? ;)

**Comment by nicolasdaudin** [comment]
Is that data feed still working? I tried to get to 'http://assoc-datafeeds-eu.amazon.com/' via my web browser and the answer is : 'File not found, please check your request and resubmit.'... Weird...

**Comment by ntzm** [comment]
@nicolasdaudin try https://assoc-datafeeds-eu.amazon.com/datafeed/listReports, you should get an authentication popup
